src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import { color, Icon, IconProps, Path, Title } from "@artsy/palette"
+
+export const DollarCircleIcon: React.FC<IconProps> = ({
+  title = "Dollar in circle",
+  ...props
+}) => {
+  return (
+    <Icon {...props} viewBox="0 0 50 50">
+      <Title>{title}</Title>
+      <Path
```

Comment:
I would recommend adding the icon to palette along with the exisitng `MoneyFillIcon` https://palette.artsy.net/atoms/icons/#gatsby-focus-wrapper

Reply:
Thanks Daria - sounds great